### DISCUSSION TITLE
Is it just me or is retrobox not yet available in neovim v0.9.1

### DISCUSSION THREAD

**Discussion by ghost** [neovim/neovim#24850] [Q&A] [answered]
Did the new colorschemes like `retrobox` and `zaibatsu` land in Neovim yet or are they just not available on my end for some reason?

I'm on Fedora right now and vim `9.0.1712` has them available. I searched for `retrobox` using `find`
and it revealed the following location of it:
```
$ sudo find / -name "*retrobox*"

/usr/share/vim/vim90/colors/retrobox.vim
```

Using `:set runtimepath?` inside of Neovim I can confirm that `/usr/share/vim/vim90/` is not part
of the runtime path. Does someone know more about it?

**Comment by clason** [comment, marked as answer]
Runtime file updates are not backported; only bugfixes. Colorschemes added after the Nvim 0.9 release will be in Nvim 0.10 (or nightly, if that fits your risk profile). You can also just copy the files from a nightly clone to your personal config to use them now.

**Reply by ghost** [reply]
Thanks for clearing that up! I will copy the file from the master branch over to my config for the time being
